### DISCUSSION TITLE
tauri v2 - Where do I configure my plugins?

### DISCUSSION THREAD

**Discussion by discoverlance-com** [tauri-apps/tauri#11288] [Q&A]
Hello, I am a bit confused on where to manage my plugins in tauri v2. Is it in the `lib.rs` or in the `main.rs`? For instance, where do I configure my `Menu`?

I tried to setup the sql plugin and by default, it updated the `lib.rs` file but when I scrolled a bit, I could see that in the examples for migrations, it was updating the `src-tauri/src/main.rs` file. So I am not sure, where do I add my plugins? By default it seems when you run the tauri command to add the plugin, it adds it to the `lib.rs` but sometimes in the docs for that plugin, there's reference to updating the `main.rs` to extend that plugin so it makes me confused as to where to manage my plugins.

**Comment by capitanhazel** [comment]
I'm having the same issue. The fact that nobody responded to this... jeez!

**Reply by FabianLars** [reply]
> The fact that nobody responded to this... jeez!

🙂

**Comment by FabianLars** [comment]
The default tauri templates should have an almost empty main.rs file that only runs the entrypoint of the lib.rs file. In general you do not need to modify the main.rs file (in tauri apps). We try to also make that clear via context, for example many examples will show the tauri::Builder part that you'll see in lib.rs

> but sometimes in the docs for that plugin, there's reference to updating the main.rs to extend that plugin so it makes me confused as to where to manage my plugins.

Would you mind sharing those? They may be a bit older than our full switch to lib.rs

**Reply by yujonglee** [reply]
for example, https://v2.tauri.app/plugin/sql/.
